test/specs/parsing/registerXQueryModule.tests.ts

```diff
@@ -266,4 +266,30 @@ declare function x:fn () external;`);
 			)
 		).to.equal('meep');
 	});
+
+	it('can propagate variables with namespaces in modules', () => {
```

Comment:
I think we could also add another unit test which does the same but have it be a bit simpler:

```js
registerXQueryModule(`module namespace foo = "${foo}";
declare variable $foo:var := "hello world";`)

fontoxpath.evaluateXPath(`import module namespace foo = "${foo}";
	$foo:var`, null, null, null, undefined, xQueryOptions);

```

Should do the exact same but without providing a namespace resolver and importing in the query itself.